app/src/main/java/io/appium/uiautomator2/model/settings/CurrentDisplayId.java

```diff
@@ -51,10 +51,11 @@ protected void apply(Integer currentDisplayId) {
             String possibleValuesMessage = TextUtils.join(",", displayIds);
 
             throw new InvalidArgumentException(String.format(
-                    "Invalid %s value specified, must be one of %s. %s was given",
+                    "Invalid %s value specified, must be one of %s. %s was given. It remains %s.",

```

Comment:
This will help to investigate the reason - whcih display id is currenly configured